### PR TITLE
GUI-2827: Implement Subnet Details page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2827

### CODE REVIEW

File: eucaconsole/forms/__init__.py [hunk 1]
```diff
@@ -474,7 +474,8 @@ def vpc_route_tables(self, vpc=None, route_tables=None, add_blank=True, escapebr
         from ..views import TaggedItemView
         choices = []
         if add_blank:
-            choices = [('None', _(u'None'))]
+            # NOTE: Do not mark the 'None' value for i18n
+            choices = [('None', _(u'None (use main route table for VPC)'))]
```
Comment (kamalgill) [inline eucaconsole/forms/__init__.py]: See the update_route_table() method in SubnetView re how we're handling this.

Comment (dkavanagh) [inline eucaconsole/forms/__init__.py]: ic. In many other places we use the empty string for this case, which is like "no selection", right?

Comment (kamalgill) [inline eucaconsole/forms/__init__.py]: We've used 'None' in other VPC resources, security groups, and key pairs, and 'None' is more explicit than '', so I stuck with that here.  I see your point about using an empty string, but we've typically used that only for the "Select..." option, which won't usually be passed across the wire due to client-side validation.

Comment (dkavanagh) [inline eucaconsole/forms/__init__.py]: That's fine, just trying to understand the usage.

**Pull request merged**